src/gui/sdlmain.cpp

```diff
@@ -4761,6 +4761,37 @@ static void init_logger(const CommandLineArguments* args, int argc, char* argv[]
 	loguru::init(argc, argv);
 }
 
+static void maybe_write_primary_config(const CommandLineArguments* args)
```

Comment:
Unless I have a good reason, I prefer to pass a reference instead of a pointer - it's safer.